app/imports/api/creature/creatureProperties/methods/insertPropertyFromLibraryNode.js

```diff
@@ -178,6 +178,7 @@ function reifyNodeReferences(nodes, visitedRefs = new Set(), depth = 0) {
     try {
       referencedNode = fetchDocByRef(node.ref);
       referencedNode.order = node.order;
+      referencedNode.tags = [...new Set(referencedNode.tags.concat(node.tags ?? []))];
```

Comment:
maybe use lodash union instead? Probably faster, definitely more readable `= union(referenceNode.tags, node.tags)`

Reply:
Updated, don't use lodash personally so wasn't aware it was an option here.